### ISSUE TITLE
Add lang_exposure to the subject aux data where applicable

### ISSUE THREAD

**Issue by adriansteffan** [langcog/peekbank-data-import#126]
Similarly to the cdi data we added in all the import scripts where applicable, let's add the language exposure to the datasets that provide the data

**Comment by adriansteffan** [comment]
I looked through the datasets, and these are still open:

- [ ] newman_genderdistractor: I extracted the messy lang exposure data from the demo, just need to import it now
- [ ] potter_remix: import needed, on it
- [ ] weisleder_stl: the paper makes it sound like there would be lang exposure data, but none is provided (or I am missing it). Any idea @mzettersten 
- [ ] There are some datasets where there is a percentage for how much english participants are exposed to(e.g. 70, 80, 100), but no values for other languages. Is it useful to import these as language exposure? (reflook_socword, frank_tablet_2016, nordmeyer_negtracker) @mzettersten
